### DISCUSSION TITLE
Collecting file from local or HDFS is too slow.

### DISCUSSION THREAD

**Discussion by pppeace** [apache/seatunnel#6304] [Q&A]
### Context
I'm trying to collect corpus(above 200g, 3000w+ rows) to Doris

### Phenomenon
I think 3~50rows/s is abnormal, whatever with localFile or HDFS.
The write/read count is close, so I guess the limit is on the read side.

### Expect
Some advices or best practice. Or maybe my requirement is not a common scenario?

### Others
And I just found an issue as same as mine.[link](https://github.com/apache/seatunnel/issues/6240), but no one help with this.😥

**Comment by pppeace** [comment]
anybody is here?

**Reply by Leeeee16** [reply]
I seem to be encountering the same issue, and I am using localfile, and mysql cdc to mysql, its too slow.

**Comment by EricJoy2048** [comment]
You need to describe the details:

seatunnel version:
jobconfig content:
submit job command:
